docs/modules/ROOT/pages/ios_files.adoc

```diff
@@ -44,9 +44,9 @@ The image below on the right, shows the move icon row for _multiple-selected_ fi
 [cols=","]
 |===
 |*Individual File Actions*
-image:file-actions.jpg[File actions dialog for individual files in ownCloud's iOS App.]
+a|image:file-actions.jpg[Individual files.]
```

Comment:
Please don't make the `alt` attribute useless. It's an important accessibility feature.